src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -1027,6 +1046,8 @@ private static FunctionResult LoadTextContentResultBuilder(IBinder binder, IFile
                 : new(ErrorType.Create(errorDiagnostic));
         }
 
+        private static IDeserializer Deserializer = new DeserializerBuilder().Build();
```

Comment:
Do you know whether this library is thread-safe? If not, I'd probably err on the side of caution and instantiate on-demand needed rather than using a `static`.